jcommune-service/src/main/java/org/jtalks/jcommune/service/nontransactional/BBCodeService.java

```diff
@@ -73,13 +74,21 @@ public String convertBbToHtml(String bbEncodedText) {
         for (TextProcessor preprocessor : preprocessors) {
             bbEncodedText = preprocessor.process(bbEncodedText);
         }
-        bbEncodedText = processor.process(bbEncodedText);
+        if (isUserbbCode(bbEncodedText)){
```

Comment:
In case when bb code contains tag [user], text processing execute with help another algoritm because kefirrbb does not work properly with eg "[user=/jcommune/users/16][user]user[/user][/user]" i.e user name contains tag [user]